bindata/network/sriov/002-rbac.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
if the service account doesn't have any RoleBindings, then you can delete all of this and the namespace's default service account will be used.